### DISCUSSION TITLE
4/13 (목) 메모리 안전

### DISCUSSION THREAD

**Discussion by Eunho0922** [Info-iOS/iOStudy#75] [General]
오늘은 스위프트의 안전을 중요시하는 것과 밀접한 관련이 있는 메모리 안전에 대해 공부해보겠습니다. 메모리 안전 개념을 잘 익히면 코드에서 위험을 줄일수 있도록 해줄 수 있습니다.

**Comment by jjunhaa0211** [comment]
헉 어려운 개념! 열심히 하겠습니다. 🥲
